File: geometry/radians_degrees_conversion/java/RadiansAndDegrees.java
import java.lang.*;

class RadiansAndDegrees{
	
	public static double degrees_to_radians(double degrees){
		return degrees * Math.PI/180;
	}

	public static double radians_to_degrees(double radians){
		return radians * 180/Math.PI;
	}

	public static void main(String[] args){
		System.out.println(radians_to_degrees(Math.PI/3));
		System.out.println(degrees_to_radians(60));
	}


	
}

File: geometry/radians_degrees_conversion/javascript/radians_and_degrees.js
// ES5 way

function radians_to_degrees(radians){
  return radians*(180/Math.PI);
}

function degrees_to_radians(degrees){
  return degrees*(Math.PI/180);
}

// ES6 way

let radians_to_degrees = (radians) => radians*(180/Math.PI);

let degrees_to_radians = (degrees) => degrees*(Math.PI/180);
